### PR TITLE
Remove deprecated languages

### PR DESCRIPTION
**Pull request by antleblanc**
| Question         | Answer
| ---------------- | ---
| Branch?          | `develop`
| Bug fix?         | yes
| New feature?     | no
| Breaking change? | yes
| Tickets          | no
| License          | BSD 3-Clause

### :boom: Breaking Change

5a72445 - refactor(plugins.translations.utils): remove deprecated languages

### :arrow_up: Upgrade

cd13dde - fix: update semver range for @ovh-ux/component-rollup-config

### :recycle: Refactor

e713bfb - refactor: refer to LANGUAGES constant outside of AngularJS

### CODE REVIEW

File: packages/manager/apps/web/client/app/app.js [hunk 1]
```diff
@@ -152,7 +152,6 @@ angular
     new_bdd_user_grant_options: config.constants.new_bdd_user_grant_options,
     ORDER_URL: config.constants.ORDER_URL,
   })
-  .constant('LANGUAGES', config.constants.LANGUAGES)
```
Comment (antleblanc) [inline packages/manager/apps/web/client/app/app.js:155]: Only used here: packages/manager/apps/web/client/app/email-domain/email/acl/email-domain-email-acl.controller.js

File: packages/manager/apps/web/client/app/config/constants.config.js [hunk 2]
```diff
@@ -741,23 +741,6 @@ module.exports = {
       contactEmail: 'validation@comodo.eu',
       phoneNumber: '+31 88 775 7777',
     },
-
-    LANGUAGES: [
```
Comment (antleblanc) [inline packages/manager/apps/web/client/app/config/constants.config.js:745]: Duplicates `LANGUAGES` from `@ovh-ux/manager-config`.

File: packages/manager/apps/web/client/app/email-domain/email/acl/email-domain-email-acl.controller.js [hunk 3]
```diff
@@ -1,3 +1,4 @@
+import { LANGUAGES } from '@ovh-ux/manager-config';
```
Comment (antleblanc) [inline packages/manager/apps/web/client/app/email-domain/email/acl/email-domain-email-acl.controller.js:1]: Prefer reuse existing list (outside of AngularJS).

**Review by mohammed-zahaf — APPROVED** [review]

**Review by cbourgois — APPROVED** [review]

File: packages/manager/modules/freefax/package.json [hunk 4]
```diff
@@ -17,11 +17,11 @@
     "ovh-manager-webfont": "^1.2.0"
   },
   "devDependencies": {
-    "@ovh-ux/component-rollup-config": "^7.0.0 || ^8.0.0"
+    "@ovh-ux/component-rollup-config": "^8.0.0 || ^9.0.0"
   },
   "peerDependencies": {
-    "@ovh-ux/manager-ng-layout-helpers": "^1.0.0 || ^2.0.0",
     "@ovh-ux/manager-core": "^11.0.0 || ^12.0.0",
+    "@ovh-ux/manager-ng-layout-helpers": "^1.0.0 || ^2.0.0",
```
Comment (marie-j) [inline packages/manager/modules/freefax/package.json:24]:
```suggestion
    "@ovh-ux/manager-ng-layout-helpers": "^2.0.0",
```
Nice catch

File: packages/manager/modules/overthebox/package.json [hunk 5]
```diff
@@ -16,11 +16,11 @@
     "lodash": "^4.17.15"
   },
   "devDependencies": {
-    "@ovh-ux/component-rollup-config": "^7.0.0 || ^8.0.0"
+    "@ovh-ux/component-rollup-config": "^8.0.0 || ^9.0.0"
   },
   "peerDependencies": {
-    "@ovh-ux/manager-ng-layout-helpers": "^1.0.0 || ^2.0.0",
     "@ovh-ux/manager-core": "^11.0.0 || ^12.0.0",
+    "@ovh-ux/manager-ng-layout-helpers": "^1.0.0 || ^2.0.0",
```
Comment (marie-j) [inline packages/manager/modules/overthebox/package.json:23]:
```suggestion
    "@ovh-ux/manager-ng-layout-helpers": "^2.0.0",
```
I think as package has now been released

**Review by marie-j — APPROVED** [review]

File: packages/manager/modules/freefax/package.json (hunk 4 above)
Comment (antleblanc) [reply packages/manager/modules/freefax/package.json:24]: Changes has been applied by `sort-package-json`.
Maybe we can handle it in a separate Pull Request (see following search results).

```sh
$ git grep "@ovh-ux/manager-ng-layout-helpers" -- "*.json"
packages/manager/apps/dedicated/package.json:    "@ovh-ux/manager-ng-layout-helpers": "^1.0.1 || ^2.0.0",
packages/manager/apps/freefax/package.json:    "@ovh-ux/manager-ng-layout-helpers": "^1.1.0 || ^2.0.0",
packages/manager/apps/hub/package.json:    "@ovh-ux/manager-ng-layout-helpers": "^1.0.0 || ^2.0.0",
packages/manager/apps/overthebox/package.json:    "@ovh-ux/manager-ng-layout-helpers": "^1.1.0 || ^2.0.0",
packages/manager/apps/sms/package.json:    "@ovh-ux/manager-ng-layout-helpers": "^1.1.0 || ^2.0.0",
packages/manager/apps/telecom/package.json:    "@ovh-ux/manager-ng-layout-helpers": "^1.0.0 || ^2.0.0",
packages/manager/modules/freefax/package.json:    "@ovh-ux/manager-ng-layout-helpers": "^1.0.0 || ^2.0.0",
packages/manager/modules/hub/package.json:    "@ovh-ux/manager-ng-layout-helpers": "^1.0.0 || ^2.0.0",
packages/manager/modules/ng-layout-helpers/package.json:  "name": "@ovh-ux/manager-ng-layout-helpers",
packages/manager/modules/ng-layout-helpers/package.json:    "start": "lerna exec --stream --scope='@ovh-ux/manager-ng-layout-helpers' --include-dependencies -- npm run build --if-present",
packages/manager/modules/ng-layout-helpers/package.json:    "start:dev": "lerna exec --stream --scope='@ovh-ux/manager-ng-layout-helpers' --include-dependencies -- npm run dev --if-present",
packages/manager/modules/ng-layout-helpers/package.json:    "start:watch": "lerna exec --stream --parallel --scope='@ovh-ux/manager-ng-layout-helpers' --include-dependencies -- npm run dev:watch --if-present"
packages/manager/modules/overthebox/package.json:    "@ovh-ux/manager-ng-layout-helpers": "^1.0.0 || ^2.0.0",
packages/manager/modules/sms/package.json:    "@ovh-ux/manager-ng-layout-helpers": "^1.0.0 || ^2.0.0",
```

Comment (antleblanc) [reply packages/manager/modules/freefax/package.json:24]: See https://github.com/ovh/manager/pull/4255

**Pull request merged**